src/main/java/org/broadinstitute/hellbender/tools/exome/ReadCountCollectionUtils.java

```diff
@@ -474,8 +474,14 @@ public double visit(final int row, final int column, final double value) {
      * <p>The imputation is done in-place, thus the input matrix is modified as a result of this call.</p>
      *
      * @param readCounts the input and output read-count matrix.
+     * @param percentile must be in [0, 100].
```

Comment:
Under normal circumstances I'd request unit tests for these new method preconditions -- but these are not normal circumstances!